assets/js/components/adminbar/AdminBarClicks.stories.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Admin Bar Clicks Component Stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { withWidgetComponentProps } from '../../googlesitekit/widgets/util';
+import { setupSearchConsoleMockReports, widgetDecorators } from './common.stories';
+import AdminBarClicks from './AdminBarClicks';
+import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
+
+const WidgetWithComponentProps = withWidgetComponentProps( 'widget-slug' )( AdminBarClicks );
+
+const Template = ( { setupRegistry, ...args } ) => (
+	<WithRegistrySetup func={ setupRegistry ? setupRegistry : () => {} }>
```

Comment:
This is a bit more concise and I'd argue a bit easier to read—`setupRegistry` appearing twice in a row is a bit tougher to parse to my eyes 🙂 
```suggestion
	<WithRegistrySetup func={ setupRegistry || () => {} }>
```

Reply:
It was tough to parse in my eyes as well, 🤣  however when i tried it, I was getting a syntax error. I went with the other solution where you suggested we set the default parameter value.